### ISSUE TITLE
Lock position of elements

### ISSUE THREAD

**Issue by HoodyEz** [excalidraw/excalidraw#8864]
I often use Excalidraw with a digital pen, and I’ve noticed that accidentally moving elements is a recurring issue. Whenever I touch elements on the interface with the pen, they tend to move unintentionally.

To address this, I suggest implementing a "Lock Position" option that prevents elements from being accidentally moved while still allowing other interactions.

This option could be added to the context menu (right-click menu), alongside the existing "Lock" feature. Unlike the current "Lock" option—which completely disables interaction with the element—this new feature would allow users to:

- Edit text blocks without risk of moving the element.
- Add text to elements or other elements in a natural workflow.
- Resize small elements intuitively without unintentional displacement.

![Image](https://github.com/user-attachments/assets/4540a0bd-3217-4c01-b5d0-585afdb653ee)

**Comment by mochsner** [comment]
This is available in obsidian-excalidraw. Not sure how it's implemented, but worth mentioning, as I thought (?) it depends on excalidraw itself.

![Image](https://github.com/user-attachments/assets/142618ce-1657-49bc-b114-839eea54c2e6)

**Comment by adamsmd** [comment]
@mochsner: The only item I see in those screenshots is "Lock", not "Lock Position".  As far as I can tell, when an element is locked, one cannot edit its text or even select the element to edit it, which is what is being asked for in this feature request.  Am I misunderstanding your suggestion?

All: This feature is also something I'd like.  I mainly want it for templates, where the template already has the text in the proper position, but the contents of the text needs to be filled out.

**Comment by HoodyEz** [comment]
> [@mochsner](https://github.com/mochsner): The only item I see in those screenshots is "Lock", not "Lock Position". As far as I can tell, when an element is locked, one cannot edit its text or even select the element to edit it, which is what is being asked for in this feature request. Am I misunderstanding your suggestion?
> 
> All: This feature is also something I'd like. I mainly want it for templates, where the template already has the text in the proper position, but the contents of the text needs to be filled out.


Yes, exactly, "lock" makes elements uneditable, while what I proposed is "lock position," which would essentially prevent the position from being altered but still allow editing of the element. This would enable actions like adding or editing text in elements such as arrows, squares, circles, etc., without accidentally moving the element. Additionally, it would allow resizing or modifying the properties of the elements without changing their current position.
